pandas/io/tests/parser/test_read_fwf.py

```diff
@@ -345,3 +345,41 @@ def test_variable_width_unicode(self):
                             header=None, encoding='utf8')
         tm.assert_frame_equal(expected, read_fwf(
             BytesIO(test.encode('utf8')), header=None, encoding='utf8'))
+
+    def test_skiprows_inference(self):
+        # GH11256
+        test = '''
```

Comment:
Heh, I was just copying the style from immediately above. :-)  Looks like the switch happens at `test_full_file`.